docker/amazonlinux.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM amazonlinux:2
+
+RUN amazon-linux-extras enable python3.8
+
+RUN yum update -y && yum install -y git make python38 gcc10 gcc10-c++ binutils tar
+
+RUN curl -L --output node.tar.gz https://github.com/nodejs/node/archive/refs/tags/v22.15.0.tar.gz
```

Comment:
It may be a good idea to verify integrity of a third-party package with a checksum, e.g.
https://github.com/async-profiler/async-profiler/blob/141ed2c72a0989fb94f749e6c7cdc515ef64f1cd/docker/debian.Dockerfile#L13-L14

Reply:
74a5f836d55e625e9c602c24209c493443fde567